### DISCUSSION TITLE
What do these characteristic peaks stand for?

### DISCUSSION THREAD

**Discussion by phantasuson** [nrc-cnrc/EGSnrc#865] [Physics]
I built a typical accelerator using BEAMnrc and generated a phase space file. These characteristic peaks are shown when I drive the electrons spectrum from the phase space file. Where these electrons came from?

![09e66cb8a1d389ab071af554189bcd9](https://user-images.githubusercontent.com/80371548/161881476-bf609bd8-0c4e-491c-971c-5c937858f10c.png)

The red line is some of the electrons inherent in the beam, and the gray one is the electrons generated on the surface of a water phantom.

**Comment by rtownson** [comment]
Hard to guess without more information. Did you use DBS in BEAMnrc? These might just be high weight ("fat") particles...

**Reply by phantasuson** [reply]
Yes I did，but I didn't split the charged particles
![bdcd4cb89515d8f7180bf0b0ee5b27d](https://user-images.githubusercontent.com/80371548/161975333-449e5166-62b9-4885-b7f9-49a5741101cb.png)

**Reply by rtownson** [reply]
The high-weight split photons can create secondary electrons who carry the same weight and those are what you're seeing.

**Reply by phantasuson** [reply]
emmm，the photon spectrum looks normal. Would that have the same effect?
![578093de9be72afd8e7ed54006f18ba](https://user-images.githubusercontent.com/80371548/161981867-452bb6fd-0b40-48fd-a515-be210f6e32e6.png)

**Reply by rtownson** [reply]
I suggest double checking that your rejection plane is above your scoring plane, and also turning on e- splitting with DBS splitting closer to 1000. This is suggested in the documentation, section 6.3 of the BEAMnrc manual.
